src/local_scheduler/local_scheduler_algorithm.h

```diff
@@ -6,9 +6,14 @@
 #include "state/local_scheduler_table.h"
 
 /* The duration that the local scheduler will wait before reinitiating a fetch
- * request for a missing task dependency. TODO(rkn): We may want this to be
- * adaptable based on the load on the local scheduler. */
-#define LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS 1000
+ * request for a missing task dependency. This time may adapt based on the
+ * number of missing task dependencies. */
+static const int64_t LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS = 1000;
+/* The duration that the local scheduler will wait between initiating
+ * reconstruction calls for missing task dependencies. If there are many missing
+ * task dependencies, we will only iniate reconstruction calls for some of them
+ * each time. */
+static const int64_t LOCAL_SCHEDULER_RECONSTRUCT_TIMEOUT_MILLISECONDS = 1000;
```

Comment:
We should use the google convention here, which is
constexpr int64_t kLocalSchedulerReconstructionTimeoutMilliseconds = 1000;